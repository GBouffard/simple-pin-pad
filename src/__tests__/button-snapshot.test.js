import React from 'react';
import Button from '../components/atoms/button';
import renderer from 'react-test-renderer';

describe('Button - ', () => {
  const component = renderer.create(
    <Button
      onClick={() => {}}
      text='this is a snapshot test' />
  );

  const tree = component.toJSON();

  it('renders correctly', () => {
    expect(tree).toMatchSnapshot();
  });
});