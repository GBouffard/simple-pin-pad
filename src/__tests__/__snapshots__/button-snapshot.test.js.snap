// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button -  renders correctly 1`] = `
<button
  className="App__pin-pad-button"
  onClick={[Function]}
>
  this is a snapshot test
</button>
`;
